ledger/participant-integration-api/src/main/scala/platform/apiserver/ApiServiceOwner.scala

```diff
@@ -68,6 +68,11 @@ object ApiServiceOwner {
       }
     }
 
+    val leewayOptions = authService match {
```

Comment:
This is equivalent to a null check after a `dynamic_cast` in c++, I would try to avoid such constructs